developer-docs-site/docs/concepts/basics-accounts.md

```diff
@@ -37,6 +37,10 @@ When a user requests to create an account, for example by using the [Aptos SDK](
 
 From now on, the user should use the private key for signing the transactions with this account. 
 
+:::tip Create an account by receiving funds
+You can also create an account by receiving funds from a third party.
+:::
```

Comment:
I worry about a few things in this block:
* Why are we pointing to the type script sdk? We aren't even pointing to a snippet of code.
* What is the goal of this to create a local representation of an account, the blockchain, or both?
* The documentation above seems to be about the local, this note seems to be about the remote.